ParkingApp/ParkingApp/Services/FileService.cs

```diff
@@ -1,43 +1,32 @@
 ﻿using Newtonsoft.Json;
 namespace ParkingApp.Services
 {
-    internal class FileService : IFileService
+    public class FileService : IFileService
     {
-        private const string Filename = "place.json";        
-
+        private const string FileName = "place.json";
+        private const string FolderName = "ParkingApp";
+       
         public Dictionary<int, ParkingPlace> DeserializeState()
         {
             var pathToRead = GetPathDirectory();
-            string result = File.ReadAllText(pathToRead);
-
-            Dictionary<int, ParkingPlace> res = JsonConvert.DeserializeObject<Dictionary<int, ParkingPlace>>(result)!;
-            //foreach (KeyValuePair<int, ParkingPlace> item in res)
-            //{
-            //    Console.WriteLine(item.Value.NumberPlace);
-            //}
-            return res;
-        }
+            string result =  File.ReadAllText(pathToRead);
 
+            Dictionary<int, ParkingPlace> resultDeserialize = JsonConvert.DeserializeObject<Dictionary<int, ParkingPlace>>(result )!;
+            return resultDeserialize;
+        }        
         public string GetPathDirectory()
         {
-            string pathFile = "";
-            var dir = Directory.GetCurrentDirectory();
-            var rr = Directory.GetParent(dir);
-
-            if (rr is not null)
+            //c:/User/users/AppData/Local
+            var dirApp = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);                  
+            string locationFile = Path.Combine(dirApp, FolderName, FileName);
+            var res = Extention.CheckPathValidCharacters(this,locationFile);
```

Comment:
this is not really needed, let's discuss

Reply:
Done.Remove usefull parts.